src/app/components/cells/attachment/AttachmentCell.jsx

```diff
@@ -22,10 +26,21 @@ const AttachmentCell = props => {
     selected: selected
   });
 
-  const openOverlay = (event, folderId) => {
-    if (isLocked(cell.row) || !editing) {
-      return;
+  const handleAttachmentLabelClick = attachmentElement => event => {
+    if (!isLocked(cell.row) && canUserChangeCell(cell, langtag)) {
+      openOverlay(event, attachmentElement.folder);
+    } else {
+      doto(
```

Comment:
[minor] Hier könnte man diesen Teil auch noch in eine Funktion "previewAttachment", "viewAttachment", "openAttachment" oder irgendwas sinnvolles  auslagern, um vom Leseflow her konsistent zu sein. Muss ja nur ein `flow(...` sein mit dem Body von `doto` ohne dem ersten Argument.